### PR TITLE
Ignore issue template files for YamlLint

### LINKED ISSUE #26628: Pull Request contain YAML linter warnings although it's out of scope
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

Every PR contains YAML linter warnings related to files in `.github/ISSUE_TEMPLATE/` even though PRs do not modify them (see screenshot)

### Expected behavior

No crazy YAML linter warnings.

### Steps to reproduce

1. Go to https://github.com/PrestaShop/PrestaShop/pull/26609/files
2. Scroll bottom.
3. See warnings.

### PrestaShop version(s) where the bug happened

develop

### PHP version(s) where the bug happened

_No response_

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Ignore issue template files for YamlLint
| Type?             | bug fix
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes [#26628](https://github.com/PrestaShop/PrestaShop/issues/26628)
| How to test?      | Refer to https://github.com/PrestaShop/PrestaShop/issues/26628
| Possible impacts? |

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26629)

### CODE REVIEW

File: .github/workflows/yamllint/.yamllint.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ extends: default
 ignore: |
   /themes/classic/_dev/node_modules/
   /themes/node_modules/
+  .github/ISSUE_TEMPLATE/
```
Comment (Progi1984) [inline .github/workflows/yamllint/.yamllint.yml:6]: Why not fixing them ?

Comment (matks) [reply .github/workflows/yamllint/.yamllint.yml:6]: Because they are not broken

**Review by tegbessou — APPROVED** [review]

File: .github/workflows/yamllint/.yamllint.yml (hunk 1 above)
Comment (atomiix) [reply .github/workflows/yamllint/.yamllint.yml:6]: I guess we could use something like that:
```YAML
value: >
        Thank you for taking the time to fill out this bug report.
        Please take the time to provide as much details as possible.
        Following our [bug report guidelines](https://devdocs.prestashop.com/1.7/contribute/contribute-reporting-issues/#best-practices-for-writing-an-issue)
        will help maintainers qualify your report faster.
```
But ignoring this file is okay for me too, I'll approve ;)

**Review by atomiix — APPROVED** [review]

File: .github/workflows/yamllint/.yamllint.yml (hunk 1 above)
Comment (matks) [reply .github/workflows/yamllint/.yamllint.yml:6]: > I guess we could use something like that:

I tried but I think the link in https://github.com/PrestaShop/PrestaShop/edit/develop/.github/ISSUE_TEMPLATE/2_feature_request.yml is still too long and cannot be split

```
[searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+)
```

**Comment by matks** [comment]
Thank you @atomiix and @tegbessou for the approval

@PrestaShop/prestashop-maintainers one more approval?

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @matks

**Pull request merged**